data/json/snippets/mutant_anatomy.json

```
@@ -43,6 +43,182 @@
+      },
+      {
+        "id": "mutant_lung_5",
+        "text": "A piece of lung from a heavily mutated animal.  The airways are lined with rows of tiny, stone-hard barbs, each pointing towards the outside, as if they were made to keep something out."
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "cooked_mutant_lung_desc",
+    "text": [
+      {
+        "id": "cooked_mutant_lung_1",
+        "text": "A cooked piece of mutant lung.  With a lot of work you can cut around the hardened shell of translucent gunk, but your only reward is getting to the spongy, grey, mostly-flavorless mass of tissue beneath."
+      },
+      {
+        "id": "cooked_mutant_lung_2",
+        "text": "A cooked piece of mutant lung.  The heat somehow made the tissue swell up, with some alveoles bursting and collapsing on themselves.  The whole thing turned to the consistency of cotton candy, only with a faint taste of yeast replacing the sugary goodness."
+      },
+      {
+        "id": "cooked_mutant_lung_3",
+        "text": "A cooked piece of mutant lung.  Deep within, the extraneus vocal cords you didn't manage to remove turned to chewy morsels.  The taste is almost good enough to make you forget the way they sounded while you were cooking it."
+      },
+      {
+        "id": "cooked_mutant_lung_4",
+        "text": "A cooked piece of mutant lung.  The glands exuding the green foam have solidified, and the foam itself seems to have evaporated without a trace.  The smell is detestable, but the glads have a taste and consistency remarkably like hardboiled egg yolk."
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "mutant_liver_desc",
+    "text": [
+      {
+        "id": "mutant_liver_1",
+        "text": "A chunk of liver from a mutated animal.  The surface is pockmarked by hexagonal scar tissue, and some surprisingly active flatworms still seem to cling to it.  About half of them seem to be in the process of being digested by their meal, instead of the other way around."
+      },
+      {
+        "id": "mutant_liver_2",
+        "text": "A chunk of liver from a mutated animal.  Most of its volume is taken up by a gigantic cyst filled with tiny, crab-like creatures.  Upon opening it the contents spray to the ground, with the suddenly homeless inhabitants scurrying for cover.  The rest of it looks relatively normal, if only in comparison."
+      },
+      {
+        "id": "mutant_liver_3",
+        "text": "A chunk of liver from a mutated animal.  The gall bladder transformed into a bundle of hair and skin, and what appeared to be brownish veins originating from it turn out to be singular, curled hairs crisscrossing beneath the surface of the organ.  At least you can remove them with a gentle pull on the central mass, but the whole liver collapses in a gelatinous heap afterwards."
```

```suggestion
        "text": "A chunk of liver from a mutated animal.  The gall bladder has transformed into a bundle of hair and skin, and what appeared to be brownish veins originating from it turn out to be singular, curled hairs crisscrossing beneath the surface of the organ.  At least you can remove them with a gentle pull on the central mass, but the whole liver collapses in a gelatinous heap afterwards."
```
"Singular" re hairs?

---

Let's try "solitary".